modules/test/playwright/tests/accessibility-menu-web/accessibilityMenu.spec.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {accessibilityMenuPagesTest} from '../../fixtures/accessibilityMenuPagesTest';
+import {instanceSettingsPagesTest} from '../../fixtures/instanceSettingsPagesTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {usersAndOrganizationsPagesTest} from '../../fixtures/usersAndOrganizationsPagesTest';
+import performLogin, {performLogout} from '../../utils/performLogin';
+import {featureFlagPagesTest} from '../feature-flag-web/fixtures/featureFlagPagesTest';
+
+const test = mergeTests(
+	accessibilityMenuPagesTest,
+	featureFlagPagesTest,
+	instanceSettingsPagesTest,
+	isolatedSiteTest,
+	loginTest(),
+	usersAndOrganizationsPagesTest
+);
+
+test('Verifies that the default value is displayed when the user has never changed the accessibility setting, regardless of sign-in status.', async ({
+	accessibilityMenuPage,
+	instanceSettingsPage,
+	page,
+	site,
+}) => {
+	await test.step('Set up accessibility settings', async () => {
+		await instanceSettingsPage.goToInstanceSetting(
+			'Accessibility',
+			'Accessibility Menu'
+		);
+
+		await accessibilityMenuPage.enableAccessibilityMenu();
+
+		await performLogout(page);
+	});
+
+	await test.step('Verify that the switch is off when logged out, then turn it on', async () => {
+		await page.goto(site.friendlyUrlPath);
+
+		await accessibilityMenuPage.displayAccessibilityMenu();
+
+		await accessibilityMenuPage.checkUnderlinedLinks();
+	});
+
+	await test.step('Log in and enable underlined links', async () => {
```

Comment:
Since "underlined links" was enabled when logged out and this is supposed to be a new user, "underlined links" should stay enabled when logged in. This test is currently not testing that.